examples/routing_service/routing_service_with_security/routingServiceProfile.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0"?>
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+    xsi:noNamespaceSchemaLocation="https://community.rti.com/schema/6.0.0/rti_routing_service.xsd">
+    <!--
+    ********************************************************************** 
+    RTI Routing Service with Security example                                            
+    ********************************************************************** 
+    This example bridges data from Domain 0 (unsecured domain) to 
+    Domain 1 (secured domain)
+    ********************************************************************** 
+    -->
+    
+    <routing_service name="security_example">
+
+        <annotation>
+            <documentation>
+                Bridges data from Domain 0 (unsecured) to Domain 1 (secured)
+            </documentation>
+        </annotation>
+        
+        <domain_route name="DomainRoute" enabled="true">
+            <!-- 
+            Participant 1 is in Domain 1, an unsecured domain 
+            -->
+            <participant name="1">
+                <domain_id>0</domain_id>
+            </participant>
+            
+            <!-- 
+            Participant 2 is in Domain 2, a secured domain 
+            -->
+            <participant name="2">
+                <domain_id>1</domain_id>
+                <!--
+                Inherit the participant QoS profile from the generic Security profile.
+                -->
+                
+                <participant_qos base_name="BuiltinQosLib::Generic.Security">
+                    <property>
+                        <value>
+                            <element>
+                                <name>dds.sec.auth.identity_ca</name>
+                                <value>file:../../../resources/certAuthority/dsa01/ca/dsa01RootCaCert.pem</value>
```

Comment:
I think this should be `file:../../../resources/cert/ecdsa01/ca/ecdsaRootCaCert.pem`

Change the rest of the values accordingly (soy they use the `ecdsa01` PKI)

Reply:
Corrected path. I have also added for `rsa01` and `dsa01`, which are commented.